docs/Configuration.md

```diff
@@ -243,7 +243,8 @@ This option requires `collectCoverage` to be set to `true` or Jest to be invoked
 
 <details>
   <summary>Help:</summary>
-  If you are seeing coverage output such as...
+
+If you are seeing coverage output such as...
```

Comment:
In MDX v2 not having a line break will wrap the summary in an extra useless paragraph. This leads browsers to ignore the nested summary tag and display  the default label instead of the provided one.

CF problem reported by Argos here: https://app.argos-ci.com/slorber/jest-website-visual-tests/builds/11/56882867

<img width="1486" alt="CleanShot 2023-09-01 at 16 45 20@2x" src="https://github.com/jestjs/jest/assets/749374/4072af9f-63d7-4233-bc9e-8655f51c2b51">

<img width="1154" alt="CleanShot 2023-09-01 at 16 45 32@2x" src="https://github.com/jestjs/jest/assets/749374/1ebb82f5-3154-4d36-8435-e8375c877fe3">

Reply:
This change can be moved to a preparation PR and would work with both MDX v1 and v2